README.md

```diff
@@ -16,13 +16,23 @@ class AdminUser < ApplicationRecord
 end
 ```
 
-
-Users will receive the following error message if they use a password from the PwnedPasswords dataset
+Users will receive the following error message if they use a password from the
+PwnedPasswords dataset:
 
 ```
 This password has previously appeared in a data breach and should never be used. Please choose something harder to guess.
 ```
 
+Optionally, you can add the following snippet to `config/initializers/devise.rb`
```

Comment:
Maybe this could be prefixed with

> By default passwords are rejected if they appear at all in the dataset.

Initially when reading this I was worried that the default would be anything buy 1 and had to check the code.

Reply:
SGTM, thanks!